src/theme/src/themes/default/foundational-styles/tokens.js

```diff
@@ -267,6 +267,8 @@ const fills = {
 // Lifted from foundational scales
 const spinnerColor = 'rgba(67, 90, 111, 0.47)'
 const overlayBackgroundColor = 'rgba(67, 90, 111, 0.7)'
+const codeBackgroundColor = 'rgba(16, 112, 202, 0.06)'
```

Comment:
Lifted from `foundational-styles.scales` (which I think that we should likely remove (!) for the `default` theme)